azure/ConsiderationsForServiceDesign.md

```diff
@@ -0,0 +1,99 @@
+## Considerations for Service Design 
+Great APIs make your service usable to customers. They are intuitive, naturally reflecting and communicating the underlying model and its behavior. They lend themselves easily to client library implementations in multiple programming languages. And they don't "get in the way" of the developer, by remaining stable and predictable, <i>especially over time</i>.
+
+This document provides Microsoft teams building Azure services with a set of guidelines that  help service teams build great APIs. The guidelines create APIs that are approachable, sustainable, and consistent across the Azure platform. We do this by applying a common set of patterns and web standards to the design and development of the API. For developers, a well defined and constructed API enables them to build fault-tolerant applications that are easy to maintain, support, and grow. For Azure service teams, the API is often the source of code generation enabling a broad audience of developers across multiple languages.
+
+Azure Service teams should engage the  Azure HTTP/REST Stewardship Board early in the development lifecycle for guidance, discussion, and review of their API. In addition, it is good practice to perform a security review, especially if you are concerned about PII leakage, compliance with GDPR, or any other considerations relative to your situation.
+
+Your goal is to create a developer friendly API where:
```

Comment:
I find it strange that the heading says "create developer friendly APIs", but then there are only two guidelines below and both talk about compat (yet there is so much more to creating dev friendly APIs). This makes me think that the two guidelines are just examples.

Reply:
Agree there are more than just two things you have to do to make a developer friendly API. It's probably worth rephrasing this a bit to reflect this and point out that we consider these two examples/items, to be core.